app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -53,37 +53,33 @@
         app:layout_constraintTop_toBottomOf="@id/lesson_thumbnail">
 
         <TextView
+          style="@style/Subtitle2"
           android:id="@+id/story_name_text_view"
           android:layout_width="match_parent"
-          android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="8dp"
           android:layout_marginEnd="8dp"
           android:ellipsize="end"
-          android:fontFamily="sans-serif"
           android:maxLines="1"
           android:text="@{viewModel.ongoingStory.storyName}"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
+          style="@style/Caption"
           android:id="@+id/topic_name_text_view"
           android:layout_width="match_parent"
-          android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="4dp"
           android:layout_marginEnd="8dp"
           android:layout_marginBottom="12dp"
           android:ellipsize="end"
-          android:fontFamily="sans-serif-light"
           android:maxLines="1"
           android:text="@{viewModel.ongoingStory.topicName}"
           android:textAllCaps="true"
           android:textColor="@color/oppiaPrimaryText"
```

Comment:
This can be removed as it is already part of `TextView.Common1` style which is the default parent style for all other styles.

Reply:
`android:textColor="@color/oppiaPrimaryText"`
You can remove this line too because this is already mentioned in styles.xml

Check this:
`Body -> TextView.Common1`